prowler/providers/kubernetes/kubernetes_provider.py

```diff
@@ -289,7 +288,7 @@ def get_all_namespaces(self) -> list[str]:
             logger.critical(
                 f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
-            sys.exit()
+            raise error
```

Comment:
Add a TODO here to replace it with our custom exceptions once they are ready.

Reply:
Done

Reply:
Done